src/gxcore/social/share/shareText.ts

```diff
@@ -0,0 +1,24 @@
+import { msg } from "../../../misc/msg";
+
+/**
+ * @param text
+ * @param url
+ * @param title
+ */
+export const shareText = (text: string, url: string, title: string) => {
+  const nav: any = window.navigator;
+  if (nav && nav.share) {
+    nav
+      .share({
+        title: title,
+        url: url,
+        text: text
+      })
+      .then(() => {
+        console.log("Thanks for sharing!");
+      })
+      .catch(console.error);
```

Comment:
Not sure if this will work properly, as you are passing the error function and, if inside the error function `this` is used, it won't be `console`.

Reply:
That code is definitely not needed, removed at [e94451b](https://github.com/genexuslabs/web-standard-functions/pull/61/commits/e94451b7ec6fbde436fbebebddfc57896e40073a)